tests/components/counter/test_init.py

```diff
@@ -56,14 +61,16 @@
 
     _LOGGER.debug("ENTITIES: %s", hass.states.async_entity_ids())
 
-    assert count_start + 2 == len(hass.states.async_entity_ids())
+    assert count_start + 3 == len(hass.states.async_entity_ids())
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=4763151)